topics/statistics/tutorials/galaxy-ludwig/tutorial.md

```diff
@@ -0,0 +1,467 @@
+<!-- You may want to cite some publications; this can be done by adding citations to the
+bibliography file (`tutorial.bib` file next to your `tutorial.md` file). These citations
+must be in bibtex format. If you have the DOI for the paper you wish to cite, you can
+get the corresponding bibtex entry using [doi2bib.org](https://doi2bib.org).
+
+With the example you will find in the `tutorial.bib` file, you can add a citation to
+this article here in your tutorial like this:
+{% raw %} `{% cite Batut2018 %}`{% endraw %}.
+This will be rendered like this: {% cite Batut2018 %}, and links to a
+[bibliography section](#bibliography) which will automatically be created at the end of the
+tutorial.-->
+
+> <agenda-title></agenda-title>
+>
+> In this tutorial, we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+
+> <comment-title>Background</comment-title>
+>
+> [MNIST dataset](https://yann.lecun.com/exdb/mnist/) provides the database of handwritten digits
+> the database has a training set of 60,000 examples, and a test set of 10,000 examples.
+> at our planet and its environment for the ultimate benefit of all European citizens.
+> It is a subset of a larger set available from NIST.
+> The digits have been size normalized and centered in a fixed-size image.
+{:  .comment}
+
+# FILES FORMAT
+Before starting our hands-on, here is a brief explanation of the three files generated for the Ludwig Experiment tool.
+
+## Image_Files.zip 
+
+Since our model will learn directly from the PNG files containing the handwritten numbers, we need to upload these files as well. The images are stored in two main folders: one for training and one for testing. Another folder, named after the figure label, is used to store the figures (Figure 2).
+
+Notice that we are going to work with compressed file (.zip), Galaxy-Ludwig knows how to decompress the files and have them ready.
+
+![folder tree related to the hand-written digits picture](../../images/galaxy-ludwig/explain_images_file.png "File tree containing the images files used to training and testing the model")
+
+## MNIST_dataset.csv
+
+The MNIST dataset consists of images and their corresponding labels. For the purpose of this tutorial, mnist_dataset.csv file is created and contains three columns: image_path, label and, split.
+
+Briefly, the image_path column provides the file paths to the images that will be fed into the deep learning algorithm. The label column contains the correct classifications, ranging from 0 to 9, for the handwritten digits in the images. The split column indicates whether the data should be used for training (0) or testing (2) the model.
+
+![Dataset.csv file format snapshot](../../images/galaxy-ludwig/explain_dataset_format.png "Dataset.csv file format snapshot. features in order: file_path, label, split.")
```

Comment:
Just as a tip, since this image is rendering perhaps larger than you want, you can add some inline styling to the end like so: (have not tested if 50% is a good value, but just as an example)

```suggestion
![Dataset.csv file format snapshot](../../images/galaxy-ludwig/explain_dataset_format.png "Dataset.csv file format snapshot. features in order: file_path, label, split."){: width="50%"}
```

Reply:
Thank you! This is great to know. I will test it and submit it with the best configuration.